library/worldgen/biome/src/main/java/org/quiltmc/qsl/worldgen/biome/api/BiomeSelectors.java

```diff
@@ -65,29 +66,31 @@ public static Predicate<BiomeSelectionContext> vanilla() {
 	}
 
 	/**
-	 * Returns a biome selector that will match all biomes that would normally spawn in the Overworld,
-	 * assuming Vanilla's default biome source is used.
+	 * Returns a biome selector that will match all biomes that would normally spawn in the Overworld.
+	 * This method is compatible with biomes defined in the {@code minecraft:is_overworld} tag.
 	 */
 	public static Predicate<BiomeSelectionContext> foundInOverworld() {
-		return context -> context.canGenerateIn(DimensionOptions.OVERWORLD);
+		return context -> context.isIn(BiomeTags.IS_OVERWORLD) || context.canGenerateIn(DimensionOptions.OVERWORLD);
 	}
 
 	/**
-	 * Returns a biome selector that will match all biomes that would normally spawn in the Nether,
-	 * assuming Vanilla's default multi noise biome source with the nether preset is used.
+	 * Returns a biome selector that will match all biomes that would normally spawn in the Nether.
+	 * This method is compatible with biomes defined in the {@code minecraft:is_nether} tag.
 	 * <p>
 	 * This selector will also match modded biomes that have been added to the nether using {@link NetherBiomes}.
 	 */
 	public static Predicate<BiomeSelectionContext> foundInTheNether() {
-		return context -> context.canGenerateIn(DimensionOptions.NETHER);
+		return context -> context.isIn(BiomeTags.IS_NETHER) || context.canGenerateIn(DimensionOptions.NETHER);
 	}
 
 	/**
-	 * Returns a biome selector that will match all biomes that would normally spawn in the End,
-	 * assuming Vanilla's default End biome source is used.
+	 * Returns a biome selector that will match all biomes that would normally spawn in the End.
+	 * This method is compatible with biomes defined in the {@code minecraft:is_end} tag.
+     * <p>
```

Comment:
i screwed up in the suggestion thing
```suggestion
   * <p>
```